tests/Stormpath/Tests/Resource/ApplicationTest.php

```diff
@@ -387,6 +393,71 @@ public function testSendPasswordResetEmailWithAccountStore()
         $groupA->delete();
     }
 
+    public function testSendVerificationEmail()
+    {
+        $application = self::$application;
+
+        $directory = \Stormpath\Resource\Directory::instantiate(array('name' => 'dir' . md5(time())));
+        self::createResource(\Stormpath\Resource\Directory::PATH, $directory);
+
+        \Stormpath\Resource\AccountStoreMapping::create(
+            array('accountStore' => $directory, 'application' => $application)
+        );
+
+        // set directory policy to enable verification email workflow
+        $policy = $directory->getProperty('accountCreationPolicy');
+        $policy->verificationEmailStatus = 'ENABLED';
+        $policyResource = new Resource(self::$client->dataStore, $policy);
+        $result = self::$client->create($policyResource->href, $policyResource);
+        $this->assertEquals('ENABLED', $result->getProperty('verificationEmailStatus'));
```

Comment:
Looks like lines 408 to 412 are making calls that are not the calls that a user would be making, for example: `$directory->getProperty('accountCreationPolicy')`. Can we change these calls.